metrix-mapping/src/main/java/com/powsybl/metrix/mapping/TimeSeriesMappingConfig.java

```diff
@@ -0,0 +1,1830 @@
+            }
+        }
+
+        checkMappedAndUnmapped(generatorToTimeSeriesMapping, unmappedGenerators, ignoredUnmappedGenerators);
+        checkMappedAndUnmapped(loadToTimeSeriesMapping, unmappedLoads, ignoredUnmappedLoads);
+        checkMappedAndUnmapped(danglingLineToTimeSeriesMapping, unmappedDanglingLines, ignoredUnmappedDanglingLines);
+        checkMappedAndUnmapped(hvdcLineToTimeSeriesMapping, unmappedHvdcLines, ignoredUnmappedHvdcLines);
+        checkMappedAndUnmapped(pstToTimeSeriesMapping, unmappedPst, ignoredUnmappedPst);
+    }
+
+    public Set<MappingKey> getEquipmentTimeSeriesKeys(Set<MappingKey> equipmentToTimeSeriesMapping, Set<MappingKey> equipmentTimeSeries) {
+        Set<MappingKey> keySet = new HashSet<>(equipmentToTimeSeriesMapping);
+        keySet.retainAll(equipmentTimeSeries);
+        return keySet;
+    }
+
+    public Set<MappingKey> getNotMappedEquipmentTimeSeriesKeys(Set<MappingKey> equipmentToTimeSeriesMapping, Set<MappingKey> equipmentTimeSeries) {
+        Set<MappingKey> keySet = new HashSet<>(equipmentTimeSeries);
+        keySet.removeAll(equipmentToTimeSeriesMapping);
+        return keySet;
+    }
+
+    public Set<MappingKey> getEquipmentTimeSeriesKeys() {
+        Set<MappingKey> keys = new HashSet<>();
+        keys.addAll(getEquipmentTimeSeriesKeys(getGeneratorToTimeSeriesMapping().keySet(), getGeneratorTimeSeries()));
+        keys.addAll(getEquipmentTimeSeriesKeys(getLoadToTimeSeriesMapping().keySet(), getLoadTimeSeries()));
+        keys.addAll(getEquipmentTimeSeriesKeys(getDanglingLineToTimeSeriesMapping().keySet(), getDanglingLineTimeSeries()));
+        keys.addAll(getEquipmentTimeSeriesKeys(getHvdcLineToTimeSeriesMapping().keySet(), getHvdcLineTimeSeries()));
+        keys.addAll(getEquipmentTimeSeriesKeys(getPstToTimeSeriesMapping().keySet(), getPstTimeSeries()));
+        keys.addAll(getEquipmentTimeSeriesKeys(getBreakerToTimeSeriesMapping().keySet(), getBreakerTimeSeries()));
+        return keys;
+    }
+
+    public Set<MappingKey> checkEquipmentTimeSeries() {
+        Set<MappingKey> keys = new HashSet<>();
+        keys.addAll(getNotMappedEquipmentTimeSeriesKeys(getGeneratorToTimeSeriesMapping().keySet(), getGeneratorTimeSeries()));
+        keys.addAll(getNotMappedEquipmentTimeSeriesKeys(getLoadToTimeSeriesMapping().keySet(), getLoadTimeSeries()));
+        keys.addAll(getNotMappedEquipmentTimeSeriesKeys(getDanglingLineToTimeSeriesMapping().keySet(), getDanglingLineTimeSeries()));
+        keys.addAll(getNotMappedEquipmentTimeSeriesKeys(getHvdcLineToTimeSeriesMapping().keySet(), getHvdcLineTimeSeries()));
+        keys.addAll(getNotMappedEquipmentTimeSeriesKeys(getPstToTimeSeriesMapping().keySet(), getPstTimeSeries()));
+        keys.addAll(getNotMappedEquipmentTimeSeriesKeys(getBreakerToTimeSeriesMapping().keySet(), getBreakerTimeSeries()));
+        return keys;
+    }
+}
+
```

Comment:
To be check: is it documented?

Reply:
This object is the result of the mapping. It is then implicitly documented with the mapping dsl documentation (https://github.com/powsybl/powsybl.github.io/pull/199/commits/07fc916f2d6948b91c2a4babc713ce822a3233b0#diff-01b633a916645a292c83d3ff2bd36428ecfa287f377b99f6ec309c0252c5241f)